Dockerfile

```diff
@@ -1,5 +1,21 @@
 FROM mhart/alpine-node:5.8.0
 
+# enable backports
+RUN echo 'deb http://deb.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/backports.list
+
+# patchy mcpatchface
+RUN apt-get update && \
+  apt-get -y dist-upgrade && \
+  apt-get clean && \
+  apt-get install -y jq python-dev python-pip && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
None of the above modifications will work as this image is based off Alpine Linux, which is not a Debian variant (no `apt-get` available). Just need to ensure `jq` and `pip` are installed.